user_manager/utils.py

```diff
@@ -0,0 +1,23 @@
+"""
+Utilities for User Manager Application.
+"""
+from __future__ import absolute_import, unicode_literals
+
+from .models import UserManagerRole
+
+
+def create_user_manager_role(user, manager_user=None, manager_email=None):
+    """
+    Create new ``UserManagerRole`` given a ``user`` and a ``manager_user`` or ``manager_email``.
+    """
+    if manager_email is not None:
+        obj, _ = UserManagerRole.objects.get_or_create(
+            unregistered_manager_email=manager_email,
+            user=user,
+        )
+    else:
```

Comment:
Should this function check here to make sure that `manager_user` is not `None` in this case?